frontend/test/metabase/scenarios/visualizations/smartscalar-trend.cy.spec.js

```diff
@@ -51,7 +51,7 @@ describe("scenarios > visualizations > scalar", () => {
     cy.icon("arrow_down");
 
     cy.get(".SmartWrapper")
-      .should("contain", "999%")
+      .should("contain", "99.9%")
```

Comment:
@flamber actually, it's most likely `99,9%`, right? Not sure the dot  will work here?

Reply:
I don't know what formatting you're using, but US style is dots for decimals.

Reply:
You are right, but it seems it'll be formatted slightly differently with two added zeroes:
![image](https://user-images.githubusercontent.com/31325167/153911901-74a026f3-2ad1-4f1b-9d4d-fa785407b22e.png)

Reply:
No, that's the opposite way than your example - either you are comparing from 1 to 1000, or from 1000 to 1.
![image](https://user-images.githubusercontent.com/1447303/153914122-5c144075-651f-48cb-9202-6796cfcc4d1b.png)